sql/hive/src/test/scala/org/apache/spark/sql/hive/StatisticsSuite.scala

```diff
@@ -116,7 +116,7 @@ class StatisticsSuite extends QueryTest with TestHiveSingleton {
     // Try to analyze a temp table
     sql("""SELECT * FROM src""").registerTempTable("tempTable")
     intercept[UnsupportedOperationException] {
-      hiveContext.analyze("tempTable")
+      AnalyzeTable("tempTable").run(hiveContext)
```

Comment:
We can use `sql("ANALYZE TABLE tempTable COMPUTE STATISTICS noscan")`. Not, no need to update the PR just for this.